templates/related-accessories.php

```diff
@@ -13,7 +19,7 @@
     <h3><?= __('Accessories & Extras', Plugin::L10N) ?></h3>
   <?php endif; ?>
 
-  <select>
+  <select <?php echo $is_notice_template && $related_accessories_groups_count <= 1 ? 'style="display: none;"' : ''; ?>>
```

Comment:
@alinistrate — any reason we are not simply not rendering this at all instead of only hiding with CSS?

Reply:
@bogdanarizancu We didn't removed it completely because there are actions set on the change event of the select. Like the init function:
```
  // Forces related accessories notice slideshow to be shown on page load.
  $('.related-accessories select').change();
```